graspologic/embed/n2v.py

```diff
@@ -142,6 +143,10 @@ def node2vec_embed(
 
     labels = list(node2vec_graph.original_graph.nodes())
     remapped_labels = node2vec_graph.label_map_to_string
+    isolated_nodes = [x for x in nx.isolates(node2vec_graph.original_graph)]
+    if len(isolated_nodes):
```

Comment:
my preference is for explicit booleanification not implicit truthiness - so `len(isolated_nodes) > 0`